tools/jil/runner/index.js

```diff
@@ -77,8 +77,14 @@ function loadDefaultFiles(cb) {
 function loadFiles(testFiles, cb) {
   for (let file of testFiles) {
     file = resolve(process.cwd(), file)
-    if (file.slice(-11) === '.browser.js' || (file.includes('/browser/') && file.endsWith('.test.js'))) {
-      loadBrowser(testDriver, file)
+    if (file.slice(-11) === '.browser.js') {
+      let spec
+      try {
+        spec = require(file.replace('.browser.', '.spec.'))
```

Comment:
This line in the runner now checks for a spec file to include in the `loadBrowser()` call that stages the browser tests. The spec file must be in the same directory as the test, and share the same name as the test (with spec.js suffix)